.github/workflows/deploy.yml

```diff
@@ -39,9 +39,4 @@ jobs:
       - name: Run Docker Container
         run: |
             docker run -e "APPLICATION_ID=${{ secrets.ALGOLIA_APPLICATION_ID }}" -e "ALGOLIA_API_KEY=${{ secrets.ALOGLIA_API_KEY }}" -e "CONFIG=$(cat config.json | jq -r tostring)" algolia/docsearch-scraper:latest
-
-      - name: Update omnibar algolia index
```

Comment:
@akash-codemonk - Should we also delete the index created on Algolia? It has no impact on the code changes done as part of the PR. But it would be good to remove the same as we no longer will be using it.

Reply:
Since the frontend of older versions will continue to be fetching the data from the index, we should probably remove it after some time. For now it is safe to keep it I think.